src/Models/GroupModel.php

```diff
@@ -4,14 +4,10 @@
 
 namespace CodeIgniter\Shield\Models;
 
-use CodeIgniter\Model;
 use CodeIgniter\Shield\Entities\User;
 
-class GroupModel extends Model
+class GroupModel extends BaseModel
```

Comment:
Isn't it better to declare the use of BaseModel?
```php
use CodeIgniter\Shield\Models\BaseModel;
```

Reply:
Why?
`BaseModel` is in the same namespace of the `GroupModel`.
So the `use` statement is not needed.